go/vt/vtctl/vtctl.go

```diff
@@ -1564,9 +1570,34 @@ func commandCreateKeyspace(ctx context.Context, wr *wrangler.Wrangler, subFlags
 	if err != nil {
 		return err
 	}
+	ktype, err := topoproto.ParseKeyspaceType(*keyspaceType)
```

Comment:
We should only call ParseKeyspaceType if keyspaceType is non-empty. Set `ktype` to NORMAL explicitly instead of relying on the value that's returned from ParseKeyspaceType along with a non-nil error. In general, it's unsafe to do anything with other return values of a function if the error returned was non-nil.